brainscore_vision/models/crossvit_pretrained/model.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+import functools
+import ssl
+
+from timm.models import create_model
+import torch
+from brainscore_vision.model_helpers.activations.pytorch import (
+    PytorchWrapper,
+    load_images,
+    preprocess_images,
+)
+from brainscore_vision.model_helpers.check_submission import check_models
+
+ssl._create_default_https_context = ssl._create_unverified_context
+
+BIBTEX = ""
+LAYERS = [
+    "blocks.1.blocks.1.0.norm1",
+    "blocks.1.blocks.1.4.norm2",
+    "blocks.1.blocks.1.0.mlp.act",
+    "blocks.2.revert_projs.1.2",
+]
+# Description of Layers:
```

Comment:
where do these come from?

Reply:
The original model submission (`cv_18_dagger_408_pretrained`) from braintree.